src/primaschema/lib.py

```diff
@@ -394,36 +394,31 @@ def build_recursive(
         build(scheme_dir=path, full=full, force=force)
 
 
+def scheme_key(scheme: dict) -> str:
+    family = scheme.get("family", "")
+    version = scheme.get("version", "")
+    return '/'.join([scheme['organism'], family, scheme['name'], version])
+
+
 def build_manifest(root_dir: Path, out_dir: Path = Path()):
     """Build manifest of schemes inside the specified directory"""
-    manifest_field_subset = [
```

Comment:
Do you want to all the fields to be carried over to the manifest? I was favouring just a subset. We don't need 10 copies of schema_version (or perhaps we do?)

Reply:
We can probably killed `schema_version` (presuming everything is kept up to date) but the more we filter the more we need to pay attention to keeping this up to date with the evolving schema.

Reply:
Ok, so maybe we exclude rather than keep fields?

Reply:
(a handful of annoying fields like schema version)

Reply:
I'll tinker add some commits to this shortly if that's ok

Reply:
Addressed in the latest commit